repo2docker/buildpacks/r.py

```diff
@@ -69,7 +69,14 @@ def detect(self):
         """
         # If no date is found, then self.checkpoint_date will be False
         # Otherwise, it'll be a date object, which will evaluate to True
-        return bool(self.checkpoint_date)
+        if bool(self.checkpoint_date):
```

Comment:
You don't need to call `bool(x)` explicitly, the `if` statement will take care of that for you.

Reply:
dah, `True` 😂